src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1512,6 +1503,88 @@ public static void InvalidJsonWithCommentsSingleSegment(string jsonString, int e
             }
         }
 
+        [Fact]
+        public static void EmptyJsonIsInvalid()
+        {
+            var dataUtf8 = ReadOnlySpan<byte>.Empty;
+            var json = new Utf8JsonReader(dataUtf8, isFinalBlock: true, new JsonReaderState());
```

Comment:
nit: Can simplify the `JsonReaderState` being passed in by just passing `default`.
`new Utf8JsonReader(dataUtf8, isFinalBlock: true, state: default);`

Reply:
here, and elsewhere